lib/r10k/forge/module_release.rb

```diff
@@ -108,12 +115,71 @@ def download
       def verify
         logger.debug1 "Verifying that #{@tarball_cache_path} matches checksum"
 
-        md5_of_tarball = Digest::MD5.hexdigest(File.read(@tarball_cache_path, mode: 'rb'))
+        contents = File.read(@tarball_cache_path, mode: 'rb')
+        sha256_of_tarball = Digest::SHA256.hexdigest(contents)
```

Comment:
One optimization is to call `Digest::SHA256.file(@tarball_cache_path).hexdigest` so that you don't have to read the entire file into memory. It should handle binary mode on Windows.

Reply:
Thanks I'll give that a shot